packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -60,26 +64,43 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
         setShouldShowPopup(false);
     };
 
+    const onHandleReturnFnDesktop = () => {
+        my_profile_store.setIsCancelAddPaymentMethodModalOpen(true);
+    };
```

Comment:
Since this function is only calling a store function, we can create it in the store but here, we can use it in the onClick only

Reply:
Got it, will refactor it asap!